src/backend/EdFi.DataManagementService.Backend.Postgresql/Operation/UpdateDocumentById.cs

```diff
@@ -35,12 +38,31 @@ NpgsqlTransaction transaction
     )
     {
         _logger.LogDebug("Entering UpdateDocumentById.UpdateById - {TraceId}", updateRequest.TraceId);
+        var documentPartitionKey = PartitionKeyFor(updateRequest.DocumentUuid);
 
         try
         {
+            DocumentReferenceIds documentReferenceIds = ReferenceHelper.DocumentReferenceIdsFrom(updateRequest.DocumentInfo.DocumentReferences);
```

Comment:
Use `using static EdFi.DataManagementService.Backend.Postgresql.ReferenceHelper;` at the top and you don't need `ReferenceHelper.` here and in your `UpsertDocument.cs` changes.